indi-svbony/svbony_ccd.cpp

```diff
@@ -219,6 +219,19 @@ bool SVBONYCCD::Connect()
         return false;
     }
 
+    // Get Camera Firmware Version
+    status = SVBGetCameraFirmwareVersion(cameraID, cameraFirmwareVersion);
+    if (status == SVB_SUCCESS)
+    {
+        LOGF_INFO("Camera Firmware Version:%s", cameraFirmwareVersion);
+    }
+    else {
+        LOG_ERROR("Error, getting Camera Firmware Version failed.\n");
```

Comment:
No need for \n

Reply:
I committed it with newline code removed form LOG_ERROR.